### DISCUSSION TITLE
Don't know where to ask this. Is PCEM forums shut down for good? What happened? Was the whole PCEMV18 fake news to begin with?

### DISCUSSION THREAD

**Discussion by KoghaandSooga** [86Box/86Box#4819] [General]
How come the forums are no more? It's just as bad as 2021 when Sarah shut her entire site down in her rage fit "I'm done this is no longer fun!' screwing the rest of the users instead of just taking a short break till she can get her bearings again and figure out a direction she wants to go, Note:  It's NOT unacceptable to take a temp vacation if your feeling stressed about a project your doing and it's going nowhere but what is unacceptable is to take it out on your fans and whip them around when the majority have done nothing to you. 

  I know your thinking 'Post on the PCEM Forums for your issue!" Now I would but of course you can't ask on PCEM forums because DUH it's shut down which is the whole purpose of this thread. 

Seriously some of the replies these days on help threads are ironic:
 About as useful as Microsoft forums when someone asks: 'But why isn't my computer turning on anymore it was working just fine yesterday? and they reply "Ummm did you try manually updating your drivers and add compatibility?" when the morons don't realize you can't even turn on the computer to get to that point!  Sheesh.

Anyways do you think the new PCEM 'maintainer' was fake just to get people to shut up and stop asking her about it? If so it seemed to have worked but not on me.

Spoiler Alert:
.  
.
.
.
.
Okay I admit she had me fooled for a little bit so shame on me I guess. Someone should do a compilation of most useless replies on a help thread over a serious computer issue. Luckily I don't suffer that on my current computer. (knocks on wood)

**Comment by dgurney** [comment]
We don't know anything more than others do, and this discussion is really not relevant here.

**Comment by OBattler** [comment]
The PCem forum appears to be up for me.
